hubgrep/lib/get_hosting_service_interfaces.py

```diff
@@ -51,20 +58,15 @@ def get_hosting_service_interfaces():
 
         SearchClass = hosting_service_interfaces_by_name[service.type]
         cache_backend = _get_cache_backend()
-        if cache_backend:
-            logger.debug(f'using cache cache backend: {cache_backend}')
-            session = CachedSession(
-                expire_after=app.config["CACHE_TIME"], backend=cache_backend
-            )
-        else:
-            session = None
+        logger.debug(f"using cache cache backend: {cache_backend}")
```

Comment:
cache cache, what? XD

Reply:
well, the cache cache. to cache the cache. 
(fixed. :P)